app/views/living_settlor/SettlorBusinessDetailsView.scala.html

```diff
@@ -0,0 +1,48 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.living_settlor.routes._
+@import models.{Mode, SettlorBusinessDetails}
+
+@this(
+    main_template: MainTemplate,
+    formHelper: FormWithCSRF
+)
+
+@(form: Form[_], mode: Mode, index : Int, draftId: String, name : String)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = s"${errorPrefix(form)} ${messages("settlorBusinessDetails.title", name)}"
+    ) {
+
+    @formHelper(action = SettlorBusinessDetailsController.onSubmit(mode, index, draftId), 'autoComplete -> "off") {
+
+        @components.back_link()
+
+        @components.error_summary(form.errors)
+
+        @components.heading("settlorBusinessDetails.heading", Some(name))
+
+        @components.input_radio(
+            field = form("value"),
+            legend = messages("settlorBusinessDetails.heading", name),
+            legendClass = Some("visually-hidden"),
```

Comment:
legendClass = Some("visually-hidden"),
Change to:
            legendClass = Some("heading-large"),
            legendAsHeading = true,